lib/functionTemplates/js/binaris.d.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="node" />
+interface BinarisHttpRequest {
+  env: Record<string, string | undefined>;
+  query: Record<string, string | string[] | undefined>;
+  body: Buffer;
+  path: string;
+  method: string;
```

Comment:
this can be more restrictive ( 'GET' | 'PUT' | ...) but I'm not sure it's valuable

Reply:
There are some non-standard headers which I assume we allow users to support via custom `HTTPResponse` such as `PATCH`.